content/the-ember-times-issue-190.md

```diff
@@ -10,19 +10,46 @@ tags:
 
 👋 Emberistas! 🐹
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+It's October, and that means its time for Hacktoberfest!
 
 ---
 
-## [🐹 1. Section title in sentence case](section-url)
+## [🎃 1. Hacktoberfest](https://hacktoberfest.digitalocean.com/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is an annual, worldwide celebration organized by DigitalOcean and sponsored by many companies that use open source software.
+Many Ember projects are participating this year! If you complete the challenge,
+you can get a prize!
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+To participate:
+
+- Sign up anytime between October 1 and October 31.
+- Make 4 approved Pull Requests to public repositories on GitHub that have the `hacktoberfest` topic.
+Take a moment to learn about what goes into an ["approved" Pull request](https://hacktoberfest.digitalocean.com/resources/participation).
+- If you are among the first 50,000 developers who complete the challenge, you get a prize. This year, you can select between a T-shirt and a tree planted in your name.
+- Join the `#hacktoberfest` channel in the [Ember Community Discord](https://discord.gg/emberjs)
+to collaborate with other participants!
+
+### Ember projects you can help out with
+
+Are you wondering what to work on?
+
+There are [81 Ember projects participating](https://github.com/search?q=topic%3Aember+topic%3Ahacktoberfest)!
+Here's a list of [recently updated issues](https://github.com/search?o=desc&q=topic%3Aember+topic%3Ahacktoberfest&s=updated&type=Repositories) in those repositories.
+
+Ember 4.0 is coming up, and so one area of need is to help addon maintainers
+upgrade to version 3.28 and resolve deprecations. Even if addon authors
+do not have issues open for this work, you can open one and offer to help.
```

Comment:
```suggestion
Ember 4.0 is coming up, and so one area of need is to help addon maintainers
upgrade to version 3.28 and [resolve deprecations](https://deprecations.emberjs.com/). Even if addon authors
do not have issues open for this work, you can open one and offer to help.
```

I wonder if upgrading an addon with deprecated patterns to 3.28 is too big of a pull request. If a large pull request doesn't get reviewed and merged immediately, the contributor's effort won't be recognized on time for Hacktoberfest.

I think we could instead suggest opening an issue to remove a deprecation or refactor code, as a concrete, small step towards upgrading an addon.

Reply:
Thank you Isaac, that’s an excellent point. I applied your suggestion.